dotnet/src/webdriver/PrintOptions.cs

```diff
@@ -106,6 +106,7 @@ public bool ShrinkToFit
         public PageSize PageDimensions
         {
             get { return pageSize; }
+            set { pageSize = value; }
```

Comment:
Or allow it to be `null`? Then we should allow `null` for other properties, and then in code consider everywhere that properties might be null to avoid `NullReferenceExceptioon`.

From my point of view allowing it to be `null` is good idea, it represents an optional parameter according specification.